Documentation/D2.md

```diff
@@ -0,0 +1,43 @@
+# Programming Hat (D2 Progress Report)
+
+## Team Members:
+* Austin Soriano
+* *Casey Key*
+* Zach Garno
+
+## Project Status
+This project is currently active. All hardware has been purchased and the Raspberry Pi Zero W (RPi) to motor switch has been assembled.
+### Impediments:
+During our first meeting we discovered that:
+ 1. The only interface to the RPi was a micro usb. We rewired a usb keyboard with a micro usb to solve this problem. As this is only necessary for manufacturing this will not be included in the cost of the final produce.
+ 2. The switch purchased required a diod which delayed initial testing. This part is now included in the modified parts list below.
```

Comment:
To manufacture a switch for the DC motor, we need a diode. This has delayed our implementation of PR feedback via DC motor. We included the diode in our modified parts list below.

Reply:
Updated.